src/client/components/dashboard/dashboard_robot/model.ts

```diff
@@ -59,15 +92,17 @@ export class DashboardRobotModel {
       lastCameraImage: 0,
       lastSeenBall: 0,
       lastSeenGoal: 0,
-      lastSeenObstacle: 0,
       penaltyReason: PenaltyReason.UNKNOWN_PENALTY_REASON,
       robotBorderColor: 'transparent',
       robotColor: '#015457',
-      robotHeading: Vector2.of(),
-      robotPosition: Vector2.of(),
+      robotPosition: Vector3.of(),
+      robotPositionCovariance: Matrix3.of(),
+      roleName: '',
```

Comment:
For the model we thought an empty string is better than unknown, that way in the view we can change it to be what we want. I was going to make it undefined but you said that would cause a problem with observables.